pkg/constants/constants.go

```diff
@@ -0,0 +1,15 @@
+package constants
+
+// constants defines some file paths that are shared outside of the
+// MCO package; and thus consumed by other users
```

Comment:
Thanks!  This should help us in the future in doing things like searching across the OpenShift repos to see which other projects are using bits from the MCO.